tests/test_prepare_graph_data.py

```diff
@@ -58,7 +57,6 @@ def test_loss(self):
         self.assertAllClose(loss_value[0], expected_loss, rtol=1e-3)
         self.assertAllClose(gs.squeeze(loss_grad), expected_grad, rtol=1e-3)
 
-    @geomstats.tests.np_and_pytorch_only
     def test_embed(self):
         """Test embedding function."""
         embeddings = self.embedding_class.embed(self.karate_graph)
```

Comment:
NIT (not so important for the tests, but more important for the core code): we should not use `embedding_class` to call the OBJECT from the class HyperbolicEmbedding, because it is precisely an object and not a class.